rotkehlchen/tests/api/test_eth2.py

```diff
@@ -784,3 +789,13 @@ def test_query_eth2_balances(rotkehlchen_api_server, query_all_balances):
     assert len(totals['assets']) == 1
     assert len(totals['liabilities']) == 0
     assert FVal(totals['assets']['ETH2']['amount']) >= amount1 + amount2 + amount3
+
+    # Eth2 and eth equivalent tests for balances
+    rotki: Rotkehlchen = rotkehlchen_api_server.rest_api.rotkehlchen
+    with patch(
```

Comment:
This test is completely wrong.

1. You are not doing an api query here, you are querying the rotkehlchen object directly.
2. You are patching the db settings instead of parameterizing as I asked you this morning in discord: 
`@pytest.mark.parametrize('db_settings', [{'include_gas_costs': True}, {'include_gas_costs': False}])  # noqa: E501`

The actual api call you can test is way above in the test at line 728. And you don't need to patch the settings. You can just call the settings endpoint and modify it as it would be in production.

Reply:
So, I tried parametrizing, but I wanted to test both False and True cases, hence I used patch. But yes, I should've used the API request, I'll fix it.